conf/application.conf

```diff
@@ -172,6 +172,7 @@ features {
     returnsEnabled = true
     paymentsEnabled = true
     deRegistrationEnabled = true
+    amendsEnabled = false
```

Comment:
shall we have this true for local for AT's ? switch off in prod

Reply:
ah yes 👍